components/go/pkg/api/api_client.go

```diff
@@ -0,0 +1,127 @@
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+
+	config "github.com/chef/chef-licensing/components/go/pkg/config"
+	// licenseclient "github.com/progress-platform-services/chef-platform-license-management/public/client/license-management/license"
+)
+
+const (
+	CLIENT_VERSION = "v1"
+)
+
+type APIClient struct {
+	URL        string
+	HTTPClient *http.Client
+	Headers    map[string]string
+}
+
+var apiClient *APIClient
+
```

Comment:
Ensure no trailing slash in the base URL. this will cause double slash issue

Reply:
Updated